### PR TITLE
Use better icons for the indicator

### PR DESCRIPTION
**Pull request by fabiozaramella**
For example, nimf-indicator icon is more recognizable now and not just a plain grey rounded square. (see below
schermata del 2017-07-13 18 36 37

### CODE REVIEW

**Comment by cogniti** [comment]
Thank you for your patch.
After fixing bugs, I will review it.

**Comment by cogniti** [comment]
Thank you, but I can not accept it. I think existing icons are better.

**Comment by fabiozaramella** [comment]
Can we discuss it? I'd like to hear why you think existing icons are better. 
Obviously it's a matter of personal preferences, and i respect that, but i also think that is important to make icons more recognizable by the user (which is the case with nimf-indicator.svg) and also to make them integrate better with most common DEs, for example the icons i proposed looks more similar to ones used by GNOME etc... and therefore integrates better.

**Comment by cogniti** [comment]
`nimf.svg` is an icon for a logo. I do not want the logo to change.

`nimf-indicator.svg` indicates the `focus-out` state. Language engine(s) should not be displayed on the icon.

Your icons are transparent icons, so they are not displayed if the background is white.
screenshot from 2017-07-16 19-36-31
screenshot from 2017-07-16 19-36-42
screenshot from 2017-07-16 19-51-24
screenshot from 2017-07-16 19-36-55

**Comment by fabiozaramella** [comment]
The fact that icons are white on the white background of file manager is perfectly normal, indicators icons are usually white svgs. The problem with nimf is that when indicators status bar changes color (for example from solid black to transparent and vice versa) they doesn't change color (from white/grey to black, from black to white/gery) to reflect the change. That's because currently nimf converts SVGs to PNGs when it should directly use SVGs for indicator icons. 

The problem also apply in some way to application icon, nimf currently uses a small svg icon and scale it up to different dimensions which is bad because it looks blurry when scaled up, and again, it should use svgs of different dimensions instead.
In conclusion, you're right about the application icon, but you should also consider not to use svgs converted and scaled too pngs but to use directly svg icons for the indicator (as in my PR) and different size svg icons for the application icon (maybe later since this icon is showed only when displaying about dialog)

**Comment by fabiozaramella** [comment]
If you prefer i'll fil two bug reports, one about using svg icons for the indicator and not pngs, one about using different size icons for the application.

**Comment by cogniti** [comment]
I have no time to create icons of different sizes.
States icons are usually opaque for compatibility. If the icons are a problem, I will add a feature to set the user icon.

**Comment by cogniti** [comment]
I've reviewed a feature to set the user icon. It needs a lot of time.
But I don’t have enough time to sleep.
I am sorry.
